CHANGELOG.md

```diff
@@ -19,14 +19,16 @@ Here is a template for new release sections
 
 ## [Unreleased]
 ### Added
-
+- Doctring of module A0 (#138)
+- Constants in `src/constants.py` (#153, #154)
 ### Changed
-- Give priority from kwargs on command line arguments (#112)
+- Give priority from kwargs on command line arguments (#112, #138)
 - Docstrings of module A1 (#113)
 - Changed keyword argument to positional argument for `create_input_json` function (#113)
-
+- function `get_user_inputs` renamed `process_user_arguments` (#138)
+- Tests for the module A0 (#138)
 ### Removed
```

Comment:
Please add 
- Terminal commands changed (#135)

Reply:
```suggestion
Terminal commands changed (#135)
### Removed
```

Reply:
@SabineHaas, you can use suggestions for this :)